plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClientImpl.kt

```diff
@@ -262,6 +269,45 @@ class AmazonQLanguageClientImpl(private val project: Project) : AmazonQLanguageC
         return CompletableFuture.completedFuture(Unit)
     }
 
+    override fun openFileDiff(params: OpenFileDiffParams): CompletableFuture<Unit> =
+        CompletableFuture.supplyAsync({
+            ApplicationManager.getApplication().invokeLater {
```

Comment:
can we follow the same pattern: https://github.com/aws/aws-toolkit-jetbrains/pull/5663#discussion_r2069777383?